### PR TITLE
front: drop simulation results and stdcm v1

### LINKED ISSUE #8060: Drop Simulation Results V1
### Description and goal

Drop all code related to simulation results V1

### Acceptance criteria

- [ ] Space time chart V1 is removed
- [ ] Speed space chart V1 is removed
- [ ] Space curve slopes chart V1 is removed
- [ ] Driver trainschedule V1 is removed
- [ ] Simulation results v1 map is removed
- [ ] Simulation results helpers are removed

### LINKED ISSUE #8238: Drop Stdcm v1
### Description and goal

Drop all code related to stdcm with V1 endpoints

### Acceptance criteria

- [ ] Related components are removed

### PR DESCRIPTION
**Pull request by SharglutDev**
close #8060 
close #8238 

This PR doesn't not handle removing V2 names, it will done in a different one.

To test : 
- simulation results v2 work properly
- synchro is still here between spaceCurvesSlopes chart, map (train moving) and the topbar with trainpositions
- stdcm debug still working
- no errors in console

### CODE REVIEW

File: front/src/modules/simulationResult/components/SpaceTimeChart/useStoreDataForSpaceTimeChart.ts [hunk 1]
```diff
@@ -1,48 +1,16 @@
...
-};
-
-export const useStoreDataForSpaceTimeChartV2 = () => {
-  const dispatch = useAppDispatch();
-
-  const infraId = useInfraID();
-
   return {
     infraId,
     trainIdUsedForProjection: useSelector(getTrainIdUsedForProjection),
```
Comment (clarani) [inline front/src/modules/simulationResult/components/SpaceTimeChart/useStoreDataForSpaceTimeChart.ts:12]: You can even remove `simulationIsPlaying` and `dispatchUpdateSelectedTrainId`

File: front/src/modules/simulationResult/components/SpaceTimeChart/useStoreDataForSpaceTimeChart.ts [hunk 2]
```diff
@@ -1,48 +1,16 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (clarani) [inline front/src/modules/simulationResult/components/SpaceTimeChart/useStoreDataForSpaceTimeChart.ts:1]: You can default export the hook I think
```suggestion
```

File: front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChart.tsx [hunk 3]
```diff
@@ -1,315 +0,0 @@
...
-        onSetChartBaseHeight(chartHeight);
-      }}
-      onResize={(_e, _dir, _refToElement, delta) => {
-        setChartHeight(chartBaseHeight + delta.height);
-        onSetChartBaseHeight(chartBaseHeight + delta.height);
-      }}
-    >
-      <div
-        id={`container-${CHART_ID}`}
-        className="speedspace-chart chart"
```
Comment (clarani) [inline front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChart.tsx:264]: You can remove the css related to the speedspace-chart and spacetime-chart

File: front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceSettings.tsx [hunk 4]
```diff
@@ -1,115 +0,0 @@
...
-      toggleSetting(SPEED_SPACE_SETTINGS_KEYS.ELECTRICAL_PROFILES);
-    }
-  }, [trainRollingStock]);
-
-  const getCheckboxRadio = useCallback(
-    (settingKey: SpeedSpaceSettingKey, isChecked: boolean, disabled?: boolean) => (
-      <CheckboxRadioSNCF
-        id={`speedSpaceSettings-${settingKey}`}
-        name={`speedSpaceSettings-${settingKey}`}
-        label={t(`speedSpaceSettings.${settingKey}`)}
```
Comment (clarani) [inline front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceSettings.tsx:82]: Some translations can be removed too (altitude for instance)

File: front/src/modules/simulationResult/components/ChartHelpers/defineChart.ts [hunk 5]
```diff
@@ -21,7 +19,6 @@ const defineChart = (
   defineY: SimulationD3Scale,
   ref: React.MutableRefObject<HTMLDivElement> | React.RefObject<HTMLDivElement>,
   rotate: boolean,
-  keyValues: ChartAxes,
```
Comment (clarani) [inline front/src/modules/simulationResult/components/ChartHelpers/defineChart.ts:24]: You can also remove the rotate props

File: front/src/modules/simulationResult/components/ChartHelpers/ChartHelpers.ts
Comment (clarani) [inline front/src/modules/simulationResult/components/ChartHelpers/ChartHelpers.ts:1]: You can remove these functions:
- offsetSeconds
- colorModelToHex

File: front/src/modules/simulationResult/components/ChartHelpers/enableInteractivity.tsx [hunk 6]
```diff
@@ -235,14 +219,10 @@ export const traceVerticalLine = (
   chart: Chart | undefined,
   keyValues: ChartAxes,
   positionValues: PositionsSpeedTimes<Date>,
-  timePosition: Date,
   rotate = false
```
Comment (clarani) [inline front/src/modules/simulationResult/components/ChartHelpers/enableInteractivity.tsx:222]: You can also remove the rotate

File: front/src/modules/simulationResult/components/SpeedSpaceChart/utils.ts
Comment (clarani) [inline front/src/modules/simulationResult/components/SpeedSpaceChart/utils.ts:1]: Move this file in `/SpaceCurveSlopes`

**Review by clarani — COMMENTED** [review]
Nice job ! 

You can also remove : 
- [x] ORSD_GRAPH_SAMPLE_DATA (in sample.ts)
- [ ] all the rotate props
- [x] the function getSimulationResults
- [x] the function onlyDigit
- [x] spaceTimeChart/types.ts

File: front/src/modules/simulationResult/components/SpaceTimeChart/useStoreDataForSpaceTimeChart.ts (hunk 1 above)
Comment (SharglutDev) [reply front/src/modules/simulationResult/components/SpaceTimeChart/useStoreDataForSpaceTimeChart.ts:12]: Done

File: front/src/modules/simulationResult/components/SpaceTimeChart/useStoreDataForSpaceTimeChart.ts (hunk 2 above)
Comment (SharglutDev) [reply front/src/modules/simulationResult/components/SpaceTimeChart/useStoreDataForSpaceTimeChart.ts:1]: Still have a warning

File: front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChart.tsx (hunk 3 above)
Comment (SharglutDev) [reply front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChart.tsx:264]: Done. Also removed the one in storybook.css.

File: front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceSettings.tsx (hunk 4 above)
Comment (SharglutDev) [reply front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceSettings.tsx:82]: Done

File: front/src/modules/simulationResult/components/ChartHelpers/ChartHelpers.ts
Comment (SharglutDev) [reply front/src/modules/simulationResult/components/ChartHelpers/ChartHelpers.ts:1]: Done

File: front/src/modules/simulationResult/components/SpeedSpaceChart/utils.ts
Comment (SharglutDev) [reply front/src/modules/simulationResult/components/SpeedSpaceChart/utils.ts:1]: Done

**Comment by SharglutDev** [comment]
> Nice job !
> 
> You can also remove :
> 
>     * [ ]  ORSD_GRAPH_SAMPLE_DATA (in sample.ts)
> 
>     * [ ]  all the rotate props
> 
>     * [ ]  the function getSimulationResults
> 
>     * [ ]  the function onlyDigit
> 
>     * [ ]  spaceTimeChart/types.ts

Done for  ORSD_GRAPH_SAMPLE_DATA, getSimulationResults, onlyDigit, spaceTimeChart/types.ts.

For all the rotates, I started to do it but I thought that if rotate is implemented in the new spacetime and speedspace charts and if we still want to keep the spaceCurvesSlopes chart, we might maybe need it ? Since it will be a lot of work to remove all of them in the chart utils and in enableInteractivity if we are not sure to drop them.

**Review by clarani — APPROVED** [review]
LGTM & tested :white_check_mark: 

It's fine for me if we keep the rotate for the moment :+1:

**Pull request merged**